docs/getting_started.md

```diff
@@ -0,0 +1,544 @@
+# The ESGF Publisher and E3SM Automated Warehouse
```

Comment:
This file combines all of the documentation into a single, unified getting starting guide.

It explains how the `esgfpub` and `warehouse` modules are related, env setup, usage guides for each module, etc.